_sass/_tables.scss

```diff
@@ -0,0 +1,12 @@
+td {
```

Comment:
I don't think all these styles should be applied on the tag selectors (`td`, `table`). It's rarely a good idea to write tag selectors unless you're writing a framework or resets. Define your own class or use Bootstrap's `table` class (if available).

Reply:
uhm.. that means i cannot use markdown table right? it would not have classes.

Reply:
Now that there's markdown syntax available, can do without these classes.